SteamKit2/SteamKit2/Steam/Handlers/SteamUser/Callbacks.cs

```diff
@@ -147,6 +153,14 @@ internal LoggedOnCallback( CMsgClientLogonResponse resp )
 
                 this.NumLoginFailuresToMigrate = resp.count_loginfailures_to_migrate;
                 this.NumDisconnectsToMigrate = resp.count_disconnects_to_migrate;
+
+                if ( resp.parental_settings != null )
```

Comment:
To the best of my knowledge this should never be `null` but at worst null data in a deserialized protobuf body, but better safe than sorry.